lighthouse-core/gather/gatherers/manifest.js

```diff
@@ -6,40 +6,18 @@
 'use strict';
 
 const Gatherer = require('./gatherer');
-const manifestParser = require('../../lib/manifest-parser');
-const BOM_LENGTH = 3;
-const BOM_FIRSTCHAR = 65279;
 
 /**
- * Uses the debugger protocol to fetch the manifest from within the context of
- * the target page, reusing any credentials, emulation, etc, already established
- * there. The artifact produced is the fetched string, if any, passed through
+ * The artifact produced is the fetched manifest string, if any, passed through
  * the manifest parser.
  */
 class Manifest extends Gatherer {
```

Comment:
any reason to keep this? TODO for followup to split up the changes nicely? Or is it worth keeping for some reason?

Reply:
boy am I regretting that 😆

Reply:
yeah I'm gonna make this a TODO